frontend/integration-tests/tests/service-catalog/service-binding.scenario.ts

```diff
@@ -19,11 +19,16 @@ describe('Test for Cluster Service Binding', () => {
     checkErrors();
   });
 
-  it('creates a new binding for new service instance `mariadb-persistent`', async() => {
+  it('creates a new binding for new service instance `jenkins-persistent`', async() => {
```

Comment:
Jenkins requires significantly more resources than many other services. It would be better to pick something lighter weight since this is going to run a lot in our merge queue.

Any reason you switched to Jenkins?

Reply:
yes, the test for csc rows was based on `icon` whereas the 'mariadb' choice uses an `img`.  I can look for another type which has an `icon`.

Reply:
Can we change the test so it doesn't require an icon?

Reply:
Jenkins should be an image, too, so I'm confused why that might fix it.

Reply:
Hmm :confused: you are right, thought the diff was img .vs icon, but parent selector is always '...icon'.  Reverted back to 'MariaDB', we'll see if it works.